docs/README.md

```diff
@@ -39,52 +39,69 @@ Other useful resources:
 
 ## Tools
 
-The default build of the docs is performed with Apache Maven.
-
 Documentation source files can be found in [Restructured
 Text](https://en.wikipedia.org/wiki/ReStructuredText) (`.rst`) format in
 `src/main/sphinx` and sub-folders.
 
 The engine used to create the documentation in HTML format is the Python-based
 [Sphinx](https://www.sphinx-doc.org).
 
-## Default build
+The [fast doc build option](#fast-doc-build-option) requires *only* a local
+installation of [Docker Desktop on
+Mac](https://docs.docker.com/docker-for-mac/install/) or [Docker Engine on
+Linux](https://docs.docker.com/engine/install/). No other tools are required.
 
-The default build is using Apache Maven and Java like for the rest of the
-Trino build. You just need to have built the current version from the root.
-Subsequently, you can build the site using the Maven wrapper script.
+The default formal build of the docs is performed with Apache Maven, which requires an
+installation of a Java Development Kit.
 
-```bash
-./mvnw -pl docs clean install
-```
+Python 3 is required for one optional way to view the generated documentation.
 
-If you have Maven installed and available on the path, you can use `mvn`
-directly.
-
-This also performs other checks, and it is the authoritative way to build the
-docs, however it is somewhat also slower than using Sphinx directly.
+## Fast doc build option
 
-## Faster build for authoring
-
-For faster local build times when writing documentation, you can run the
-Sphinx build directly. The build runs inside a Docker container and thus
-does not require having anything installed locally (except for Docker):
+For fast local build times when writing documentation, you can run the Sphinx
+build directly. The build runs inside a Docker container and thus does not
+require having anything installed locally other than Docker. You can run the
+Sphinx build on a fresh clone of the project, with no prerequisite commands. For
+example:
 
 ```bash
 docs/build
 ```
 
-Sphinx will attempt to perform an incremental build, but it does not work
+Sphinx attempts to perform an incremental build, but this does not work
 in all cases, such as after editing the CSS. You can force a full rebuild
 by doing a Maven clean first:
 
 ```bash
 ./mvnw -pl docs clean
+./docs/build
```

Comment:
./ is not needed I think

Reply:
I'm so used to being cd'd to docs, I always type ./build. Fixing.

Reply:
Well, no, we removed that line in favor of rm -rf docs/target/html